c2c-extension-examples/examples/c2c-deployment/c2c_deployment.out

```diff
@@ -0,0 +1,82 @@
+# In order to override the default values given by the compiler, we’ll be making Ballerina.cloud file in the project directory. This follows toml rules. We’ll use the following values for simplicity. All the supported key value properties can be found in code to cloud specification.
+
+[container.image]
+repository="wso2inc"
+name="hello"
+tag="v0.1.0"
+
+# Note that ballerina build should be executed in the project directory. Code to Cloud generates only one container per project.
+$ ballerina build -a
+
+Compiling source
+        wso2/hello:0.1.0
+
+Creating balos
+        target/balo/hello-2020r2-any-0.1.0.balo
+
+Running Tests
+
+        wso2/hello:0.1.0
+        No tests found
+
+Generating executables
+        target/bin/hello.jar
+
+Generating artifacts...
+
+        @kubernetes:Service                      - complete 1/1
+        @kubernetes:Deployment                   - complete 1/1
+        @kubernetes:HPA                          - complete 1/1
+        @kubernetes:Docker                       - complete 2/2
+
+        Execute the below command to deploy the Kubernetes artifacts:
+        kubectl apply -f /home/wso2/test/c2c-slp6/target/kubernetes/hello
+
+        Execute the below command to access service via NodePort:
```

Comment:
```suggestion
        Execute the command below to access the service via NodePort:
```

Reply:
This is the output from ballerina compilation. I think we should fix it there before fixing it here